### DISCUSSION TITLE
Notifications page bug

### DISCUSSION THREAD

**Discussion by HRxiaohu** [community/community#28458] [Other Feature Feedback, Questions, & Ideas] [answered]
I once forked a repo, but it is now disabled by GitHub, I can't set the notification in the notification page to read, Here is the picture.
![image](https://user-images.githubusercontent.com/63327671/183051548-1c98f5b3-09aa-4a61-8dc1-8ed7f306b320.png)

**Comment by radar-blip** [comment]
Your notification light staying on?

**Comment by xRodzMC** [comment]
![Screenshot_2022-08-05-18-07-13-85_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/100104970/183055142-da118f17-4414-482e-8f13-21be30092c81.jpg)
Make sure you are subscribed to a repository, or check your access your [notification settings here](https://github.com/settings/notifications)
![Screenshot_2022-08-05-18-06-23-75_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/100104970/183055016-ab0a2d31-69fb-4d40-80e1-031ea770b956.jpg)

> **Note**
> If you found this helpful, tap mark as answer <3

**Comment by HRxiaohu** [comment]
Yes.



---- Replied Message ----
| From | David ***@***.***> |
| Date | 08/05/2022 18:00 |
| To | ***@***.***> |
| Cc | ***@***.******@***.***> |
| Subject | Re: [community/community] Notifications page bug (Discussion #28458) |

Your notification light staying on?

**Comment by radar-blip** [comment, marked as answer]
Ok, mine is too. Must be a bug.
You can send a bug report here: https://support.github.com/contact/bug-report
